vispy/visuals/colorbar.py

```diff
@@ -593,6 +593,7 @@ def label(self):
 
     @label.setter
     def label(self, label):
+        assert()
```

Comment:
regression on my part. Will fix.

I'll make a separate PR adding a new test for this - this regression should have been caught, I'm not sure how it wasn't.